backend/routes.js

```diff
@@ -29,7 +32,11 @@ const configureRoutes = ({ oauthApi, manageUsersApi, nomisUsersAndRolesApi }) =>
   router.use('/manage-external-users/:userId', manageAuthUserRouter({ oauthApi, manageUsersApi }))
   router.use('/manage-dps-users/:userId', manageDpsUserRouter({ oauthApi, nomisUsersAndRolesApi, manageUsersApi }))
   router.use('/manage-groups', manageGroupsRouter({ manageUsersApi }))
-  router.use('/manage-roles', manageRolesRouter({ oauthApi, manageUsersApi }))
+  router.use('/manage-roles', manageRolesRouter({ manageUsersApi }))
+  router.use('/email-domains', manageEmailDomainsRouter({ manageUsersApi }))
+  router.use('/create-email-domain', createEmailDomainsRouter({ manageUsersApi }))
+  router.use('/delete-email-domain?id=:domainId&name=:domainName', deleteEmailDomainsRouter({ manageUsersApi }))
+  router.use('/delete-email-domain', deleteEmailDomainsRouter({ manageUsersApi }))
 
```

Comment:
Do we need both for delete or is the router clever enough to know they're the same (I don't know without testing)

Reply:
Yes, I do need both.